src/__tests__/CompareResults/CompareResultsTable.test.tsx

```diff
@@ -463,7 +463,7 @@ describe('Compare Results Table', () => {
 
     popper = await waitFor(() => screen.getByRole('tooltip'));
     
-    expect(popper).toHaveAttribute('style', '');
+    expect(popper).toHaveAttribute('style', 'position: absolute; top: 0px; left: 0px; margin: 0px; right: 0px; transform: translate(0px, 0px);');
```

Comment:
What is this change doing?

Reply:
This change was also suggested on the logs...a test was failing because the expected value expect(popper).toHaveAttribute('style', 'position: absolute; top: 0px; left: 0px; margin: 0px; right: 0px; transform: translate(0px, 0px);');  did not match the current value expect(popper).toHaveAttribute('style', '');

I will look into whether I can make the tests pass without the above changes.